Gemfile

```diff
@@ -1,4 +1,4 @@
-source 'https://rubygems.org'
+source 'http://rubygems.org'
```

Comment:
As I recall, I was getting some kind of error when trying to run bundle install that indicated I should remove the 's.' (Sorry, I didn't document the error I was encountering, so I can't remember what exactly it was.) I was able to proceed as normally after following that instruction. But I should have changed it back afterward. (In fact, I just did so in my local repo, and bundle install is still working properly.)